test/test_parse_eu.py

```diff
@@ -0,0 +1,231 @@
+#
+# Copyright 2017 Mycroft AI Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from datetime import datetime
+import unittest
+
+from lingua_franca.parse import (normalize, extract_numbers, extract_number,
+                                 extract_datetime, extract_datetime_eu,
+                                 isFractional_eu)
```

Comment:
ah these, eu specific functions should get imported directly `from lingua_franca.lang.parse_eu`

Though if the language is set you _should_ be able to just call `extract_datetime()`

Reply:
Fixed for extract_datetime_eus.

For isFractional_eu, the problem seems to be that the function only exists for some languages (cs, pl and now eu) and it is not in parse.py. The problem has not arisen before because the other two languages have no tests for it... So I think I shall leave this as it is.